### PR TITLE
Add Names and DataVariables for pointers in __objc_classrefs

### LINKED ISSUE #12: Apply void * type to all within-image pointers in __objc_classrefs
Could also potentially rename the datavariables.

Manually defining these results in the analysis improvement: 

image
image

to (ignore anything other than the pointer in this screenshot I didn't have my PR fork loaded for the first two but the analysis is the same)

image

### LINKED ISSUE #14: Annotate class references
Class references in the `objc_classrefs` section should be annotated/named. Right now, they aren't.

### PR DESCRIPTION
**Pull request by cxnder**
Resolves #12

### CODE REVIEW

File: Core/Analyzers/ClassRefAnalyzer.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+}
+
+void ClassRefAnalyzer::run()
+{
+    const auto sectionStart = m_file->sectionStart("__objc_classrefs");
+    const auto sectionEnd = m_file->sectionEnd("__objc_classrefs");
+    if (sectionStart == 0 || sectionEnd == 0)
+        return;
+
+    // TODO: Dynamic Address size for armv7
```
Comment (jonpalmisc) [inline Core/Analyzers/ClassRefAnalyzer.cpp:18]: Does this need to be resolved prior to merging?

File: Core/Analyzers/ClassRefAnalyzer.h [hunk 2]
```diff
@@ -0,0 +1,18 @@
+#pragma once
+
+#include "../Analyzer.h"
+
+namespace ObjectiveNinja {
+
+/**
+* Analyzer for extracting Objective-C class information.
+ */
```
Comment (jonpalmisc) [inline Core/Analyzers/ClassRefAnalyzer.h:9]: Formatting is weird here. Please run `clang-format -i --style=WebKit` on all of the files touched by this PR.

File: InfoHandler.cpp [hunk 3]
```diff
@@ -225,6 +230,20 @@ void InfoHandler::applyInfoToView(SharedAnalysisInfo info, BinaryViewRef bv)
         defineSymbol(bv, ci.methodListAddress, ci.name, "ml_");
     }
 
+    for (const auto classRef : info->classRefs)
+    {
+        DataVariable variableForClassRef;
```
Comment (jonpalmisc) [inline InfoHandler.cpp:235]: This looks unused?

File: InfoHandler.cpp [hunk 4]
```diff
@@ -235,4 +254,5 @@ void InfoHandler::applyInfoToView(SharedAnalysisInfo info, BinaryViewRef bv)
     log->LogInfo("Found %d classes, %d methods, %d selector references",
         info->classes.size(), totalMethods, info->selectorRefs.size());
     log->LogInfo("Found %d CFString instances", info->cfStrings.size());
+    log->LogInfo("Found %d Classrefs", info->classRefs.size());
```
Comment (jonpalmisc) [inline InfoHandler.cpp:257]: Please change to `"Found %d class references"`

**Review by jonpalmisc — CHANGES_REQUESTED** [review]
Looks good barring a few minor changes.

File: Core/Analyzers/ClassRefAnalyzer.cpp (hunk 1 above)
Comment (cxnder) [reply Core/Analyzers/ClassRefAnalyzer.cpp:18]: No, it requires a later larger refactor to several portions of the tool, and this is just making a note at one of the many places that will have to be updated.

**Review by jonpalmisc — APPROVED** [review]

**Pull request merged**